core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/RenderingTests.kt

```diff
@@ -106,23 +117,121 @@ class RenderingTests : JupyterReplTestCase() {
     private fun JsonArray<*>.getObj(index: Int) = this.get(index) as JsonObject
 
     @Test
-    fun `test kotlin notebook plugin utils groupby`() {
-        @Language("kts")
```

Comment:
let's keep these again, or actually even better, move it to executeAndParse(@Language("kts") script: String)